grouper/fe/settings.py

```diff
@@ -22,6 +22,7 @@ def override_timezone(self, value):
     "date_format": "%Y-%m-%d %I:%M %p",
     "cdnjs_prefix": "//cdnjs.cloudflare.com",
     "user_auth_header": "X-Grouper-User",
+    "refresh_interval": 60,
```

Comment:
This is just the default. Settings are maintained in the config file, and we can set this to whatever we want there. In prod we set it to 10s right now, but could do w/e.